src/components/Table/GridTable.tsx

```diff
@@ -901,7 +845,12 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
             ? rowClickRenderFn(as)
             : defaultRenderFn(as);
 
-        return renderFn(idx, cellCss, content, row, rowStyle);
+        let rendered = renderFn(idx, cellCss, content, row, rowStyle);
```

Comment:
**thought (non-blocking):** There is so much going on here that it'd be nice if the `idx` value was more contextual, like maybe `colIdx` just so people quickly know this is the index of the column vs row.

Reply:
Granted, you can look back at the `columns.map` and see what the idx is referencing.